ChangeLog

```diff
@@ -105,6 +105,11 @@
 	* Added lemmas `foldrE`, `foldlE`, `foldl_idx` and `sumnE`
 	  to turn "seq statements" into "bigop statements"
 
+	* Added allpairs_dep, the dependent version of allpairs,
+	  with the same notation `[seq E | i <- s, j <- t]`
```

Comment:
As `all pairs_dep` is now a secondary definition, so this should be rephrased to emphasize that the notation is now central, and `allpairs` is a self-expanding definition.

Reply:
done